### PR TITLE
details for required_env docs

### PR DESCRIPTION
**Pull request by grosser**
@adammw 

... does this make sense ?

### CODE REVIEW

File: plugins/kubernetes/README.md [hunk 1]
```diff
@@ -67,7 +67,8 @@ environment variables, secrets, etc and then sends them to the Kubernetes API.
 Multiple roles will likely be similar, but have different commands or liveness probes.
 ([kucodiff](https://github.com/grosser/kucodiff) can be used to make sure they stay in sync).
 
-Validate required environment variables are set by adding `metadata.annotations.samson/required_env` (space separated).
+Validate required environment variables are set by adding `metadata.annotations.samson/required_env` (space separated) 
+to any template (what includes `spec.containers`).
```
Comment (adammw) [inline plugins/kubernetes/README.md]: I would have said.. to your pod definition. (For example, for a deployment `spec.template.manifest.annotations.samson/required_env`)

Comment (adammw) [reply plugins/kubernetes/README.md]: the any template sounds a little odd

**Comment by grosser** [comment]
updated

File: plugins/kubernetes/README.md [hunk 2]
```diff
@@ -67,7 +67,8 @@ environment variables, secrets, etc and then sends them to the Kubernetes API.
 Multiple roles will likely be similar, but have different commands or liveness probes.
 ([kucodiff](https://github.com/grosser/kucodiff) can be used to make sure they stay in sync).
 
-Validate required environment variables are set by adding `metadata.annotations.samson/required_env` (space separated).
+Validate required environment variables are set by adding `metadata.annotations.samson/required_env` (space separated)
```
Comment (adammw) [inline plugins/kubernetes/README.md]:
```suggestion
Validate required environment variables are set by adding `metadata.annotations.samson/required_env` to the pod definition with a space separated list of variable names.
```

**Review by adammw — APPROVED** [review]

**Pull request merged**